pymedphys/_mosaiq/constants.py

```diff
@@ -25,12 +25,48 @@
 
 
 FIELD_TYPES = {
+    0: "Unspecified",
     1: "Static",
     2: "StepNShoot",
+    3: "Setup",
+    4: "kV Setup",
     5: "CT",
     6: "Port",
+    7: "Fixed",
+    8: "Dynamic",
+    9: "MV CT",
     11: "Arc",
     12: "Skip Arcs",
     13: "VMAT",
     14: "DMLC",
+    15: "Helical",
+    16: "Fixed Angle",
+    17: "Path",
+    18: "Shot",
+    20: "User Defined",
+    21: "PDR",
+}
+
+TOLERANCE_TYPES = {
```

Comment:
Just remembered this table will vary from clinic to clinic, don't need to include this into the master

Reply:
Yeah, you'll need to think of a way to separate those tolerance types into your own custom centre logic.

Reply:
@rembishj can you delete this one.